### ISSUE TITLE
Can we expect a Fix for also ripping sound on reddit?

### ISSUE THREAD

**Issue by Gramrock** [RipMeApp/ripme#1947]
* Ripme version: 1.7.95
* Java version: 1.8.0_291
* Operating system: Windows 10 Pro

* Exact URL you were trying to rip when the problem occurred: https://www.reddit.com/r/cats/
* Please include any additional information about how to reproduce the problem:

## Expected Behavior

Videos should have sound

## Actual Behavior

Videos do not have sound.

The DASH_Audio track has to be included but I am not a coder but I read it on several posts now. Is there a way we can do that?
Plz help

**Comment by cyian-1756** [comment]
The easiest fix would be switching to a program like jdownloader/youtube-dl for downloading video from reddit

**Comment by Gramrock** [comment]
> The easiest fix would be switching to a program like jdownloader/youtube-dl for downloading video from reddit

Yeah the problem is that I can't find any help on how to use it for reddit but I read that it can also bulk download subreddits contents with filter and such, but as i said, no way i could make that work, all examples and tutorials are for downloading youtube playlists and such...

And I just installed Jdownloader but that doesnt seem to have any options like ripme, or at least simple options like downloading multiple videos from a subreddit but some download filters applied... What an u missing there? can Jdownloader download subreddits videos with sound? I am looking for a script that works like ripme but where the sound track either gets downloaded and merged witht he video automatically or where i at least get the audio file as a separate file at least...
